public/index.html

```diff
@@ -1,54 +1,124 @@
-              </div>
-            </li>
-          </ul>
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="format-detection" content="telephone=no">
+  <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, height=device-height, target-densitydpi=device-dpi">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+  <link rel="stylesheet" href="css/index.css">
+  <title>Payments Example</title>
+</head>
+
+<body>
+  <div class="container">
+    <h1><a name="top"></a>Mozilla Payments</h1>
+    <nav class="navbar navbar-inverse">
+      <div class="collapse navbar-collapse">
+        <ul class="nav navbar-nav">
+          <li role="presentation"><a href="#quick-example">Quick Example</a></li>
+          <li role="presentation"><a href="#features">Features</a></li>
+          <li role="presentation"><a href="#drop-in">Drop-in UI</a></li>
+          <li role="presentation"><a href="#full-api">Full API</a></li>
+          <li role="presentation"><a href="#management">Management</a></li>
         </ul>
       </div>
-      <!--
-      <div id="your-products">
-        <h2>Purchases</h2>
-        <ul>
-          <li class="placeholder product">You haven't purchased anything yet.</li>
-        </ul>
-      </div>
-      -->
-      <div>
-        <button id="manage">Manage your account</button>
-      </div>
-    </div>
-    <footer>
-      <a target="_blank" href="https://github.com/mozilla/payments-example">source code</a>
-    </footer>
+    </nav>
+  </div>
+
+  <div class="callout">
+    <p><a href="https://payments.readthedocs.org">Mozilla Payments</a> is a unified payments interface to Mozilla sites and properties. We provide a wrapper around <a href="https://www.braintreepayments.com/">Braintree</a> (or possibly other payments providers).
+      Consider us a replacement for using Stripe, Braintree and others.</p>
+    <p>Work at the Mozilla Foundation or Mozilla Corporation and want to accept payments? This is for you.</p>
+  </div>
+
+  <div class="container">
+    <section>
+      <h2><a name="quick-example"></a>Quick Example</h2>
+      <p><a href="#top">↑ Back to top</a></p>
+      <p>We set up a sample an example product called "Mozilla Concrete". You can buy some for $10.</p>
+      <button type="button" class="btn btn-lg btn-success brick">Buy</button>
+    </section>
+
+    <section>
+      <h2><a name="features"></a>Features</h2>
+      <p><a href="#top">↑ Back to top</a></p>
+      <ul>
+        <li>Authentication through Firefox Accounts</li>
+        <li>Subscriptions</li>
+        <li>Management interface</li>
+      </ul>
+      <p>Todo list:</p>
+      <ul>
+        <li>One off purchases</li>
+        <li>Anonymous purchases</li>
+        <li>Paypal</li>
+      </ul>
+    </section>
+
+    <section>
+      <h2><a name="drop-in"></a>Drop-in UI</h2>
```

Comment:
Maybe we could call this something different. E.g. payments client? I thought this mean the braintree drop-in UI initially.